src/main/java/uk/gov/hmcts/reform/logging/layout/ReformLoggingLayout.java

```diff
@@ -37,6 +42,26 @@ public void setRequireAlertLevel(boolean requireAlertLevel) {
         this.requireAlertLevel = requireAlertLevel;
     }
 
+    public void setRequireErrorCode(boolean requireErrorCode) {
+        this.requireErrorCode = requireErrorCode;
+    }
+
+    private void appendExtraExceptionFlags(StringBuilder log, ILoggingEvent event) {
```

Comment:
it would be nice to have all the public methods first.

Reply:
do we have this as preferred guide? there are two ways:
- as you suggest. which is basically for interface kind of approach
- grouping related methods to make the code more readable, easier to understand. get the idea of class split possibilities

could move one method down as there's no more. was just having this idea that the actual method usage firstly needs to be defined 😄